phpstan-baseline.neon

```diff
@@ -1680,6 +1680,11 @@ parameters:
 			count: 1
 			path: lib/CodeTransform/Tests/Adapter/WorseReflection/Refactor/WorseGenerateMethodTest.php
 
+		-
+			message: "#^Method Phpactor\\\\CodeTransform\\\\Tests\\\\Adapter\\\\WorseReflection\\\\Refactor\\\\WorseGenerateMutatorTest\\:\\:provideExtractMutator\\(\\) has no return type specified\\.$#"
```

Comment:
you can remove this and use the `generate docblock` code action to add the docblock (might need tweaking though)

Reply:
done :)